specs/governed-transitions/chunks/chunk-02-backend-engine/context.md

```
@@ -0,0 +1,264 @@
+- `demo_gt_journal.source` is a `v.object({ channel: v.string(), actorId: v.optional(v.string()), ... })`
+
+## Convex Conventions
+
+- Use `mutation` from `./_generated/server` (NOT the fluent builder)
+- Use `v` from `"convex/values"` for argument validators
+- Use `internal` from `./_generated/api` for scheduling internal functions
+- The `executeEffect` and `hashChainJournalEntry` internal functions will be created in chunk-03. For now, include their references in the transition engine (they'll be `internalMutation` exports in the same file). If TypeScript complains about missing exports, stub them as empty `internalMutation`s.
+
+## File Structure
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)